services/cd-service/pkg/repository/repository.go

```diff
@@ -727,7 +727,31 @@ func (r *repository) ProcessQueueOnce(ctx context.Context, e transformerBatch, c
 				return
 			}
 			// Apply the items
-			transformerBatches, err, changes = r.applyTransformerBatches(transformerBatches, false, nil)
+			if r.DB.ShouldUseEslTable() {
```

Comment:
Maybe move fetch in the beginning of applyTransformerBatches so that we don't have to repeat code